app/routes/posts/form.jsx

```diff
@@ -0,0 +1,85 @@
+import { redirect } from "@remix-run/node"
+import {useState} from 'react'
+
+export const action = async ({request}) => {
+  const form = await request.formData();
+  const name = form.get('first-name')
+  const email = form.get('email')
+  const inPerson = form.get('in-person')
+  const online = form.get('online')
+  const pizza = form.get('pizza')
+
+  const fields = {name, email, inPerson, online, pizza}
+
+  const post = await db.post.create({data: fields})
+  
+  return redirect('/')
+}
+
+export default function Forms(){
+
+  const [isChecked, setIsChecked] = useState(false)
+
+
+function handleClick(e){
```

Comment:
Idiom of programming - Naming things is hard

When I encounter a `handle` function I need to think about what it is handling. Here `handleClick` doesn't help me know what is being clicked. Also notice that the actual event as referenced below is a change, not a click.

I would go with something that better explains what the function is meant to do. This gets very useful when you get into larger forms where you have a lot of handlers.

```jsx
function handleChangeAttendance(evt) {}
```
Even better would be to have the function named the same as the input name itself. So the logic would break down to `handle{eventType}{inputName}` other variations can be to swap the type and name, ie handleAttendanceChange. 

Everyone has opinions on naming things, from function names, variables (no matter the scope), components, and even names on inputs. In the end naming things only helps you and your team. Your end users get minified code anyway so they would never know the difference.